mkdocs.yml

```diff
@@ -52,6 +52,8 @@ nav:
   - Home: index.md
   - Get Started: get-started.md
   - Downloads: downloads.md
+  - Updates:
+    - October 2018: updates/2018-10.md
```

Comment:
Thought about it this morning, do we want to name them <Month> <Year>, how do we handle the scenario where we want to do additional updates/announcments? :)